test/unit/worker_test.js

```diff
@@ -27,10 +27,11 @@ describe('Workers', () => {
       passedCount += 1;
     });
 
-    workers.on(event.all.result, (status) => {
+    workers.on(event.all.result, (status, _, stats) => {
+      console.log(stats);
       expect(status).equal(false);
-      expect(passedCount).equal(4);
-      expect(failedCount).equal(2);
+      expect(passedCount).equal(5);
```

Comment:
not sure why did you update test result here...

Reply:
A new test is added to it before this change got merged, so passing test count is 5 now